src/web/theme/pages/TryIt/TryIt.js

```diff
@@ -1,9 +1,78 @@
-import React from "react";
+import React, { Fragment } from "react";
+import PropTypes from "prop-types";
+import { compose } from "recompose";
+import EnhanceTryIt from "./EnhanceTryIt";
+import Button from "theme/ui/atoms/Button";
 
-const TryIt = () => {
-  return <div>Work In Progress</div>;
+import "./TryIt.scss";
+import SelectStore from "./SelectStore";
+
+const steps = ["When", "Where", "Why"];
+
+let step = ({
```

Comment:
Why `let` ? :confused: (same below)

Also I think this could be implemented a bit more cleanly by associating the steps with some sort of render function directly  (not sure if this makes sense)